### ISSUE TITLE
ch-image: can’t pull “scratch”

### ISSUE THREAD

**Issue by qwofford** [hpc/charliecloud#1013]
In version 0.22, I'm having trouble build a base image from scratch:

```
bash-4.2$ ch-image build -t scratch-test -f ./Dockerfile .
  1 FROM scratch 
manifest: downloading
error: GET failed; expected status {200} but got 404: Not Found
```

**Comment by reidpr** [comment]
For context, `scratch` is the blank image. It should work. I guess it needs special handling.

**Comment by reidpr** [comment]
Fixed by PR #1037.
